### PR TITLE
Test PR for hostNetwork

### PR DESCRIPTION
**Pull request by mjsoyeon**
- downgrade k8s version 1.22 to 1.20

**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**

**What testing is done?**

### CODE REVIEW

File: hack/kops-patch.yaml [hunk 1]
```diff
@@ -8,8 +8,13 @@ spec:
...
-            "elasticfilesystem:DescribeAccessPoints"
+            "elasticfilesystem:DescribeAccessPoints",
+            "elasticfilesystem:DescribeMountTargets",
+            "ec2:DescribeAvailabilityZones"
           ],
           "Resource": "*"
         }
       ]
+  cloudConfig:
+    awsEBSCSIDriver:
```
Comment (Ashley-wenyizha) [inline hack/kops-patch.yaml:19]: `awsEFSCSIDriver`?

File: hack/e2e/kops.sh [hunk 2]
```diff
@@ -54,6 +54,7 @@ function kops_create_cluster() {
     fi
     if test -f "$KOPS_PATCH_NODE_FILE"; then
       kops_patch_cluster_file "$CLUSTER_FILE" "$KOPS_PATCH_NODE_FILE" "InstanceGroup" "Node"
+      kops_patch_cluster_file "$CLUSTER_FILE" "$KOPS_PATCH_NODE_FILE" "InstanceGroup" "Master"
```
Comment (Ashley-wenyizha) [inline hack/e2e/kops.sh:57]: Why would we need both Node & Master here?

Comment (mjsoyeon) [reply hack/e2e/kops.sh:57]: This part was discarded in this pr https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1088/files

**Pull request closed without merging**